src/client/layouts/ConsentsLayout.tsx

```diff
@@ -59,6 +60,9 @@ export const ConsentsLayout: FunctionComponent<ConsentsLayoutProps> = ({
         css={form}
         action={`${Routes.CONSENTS}/${page}${returnUrlQuery}`}
         method="post"
+        onSubmit={(e) => {
```

Comment:
can `e` be changed to `event` ?

Reply:
Destructuring `e` to `{ target: form }`, so that the `form` variable is the `event.target` which is what we're interested in

Reply:
and passing the `form` variable like so:

```ts
onboardingFormSubmitOphanTracking(
            page,
            pageData,
            form as HTMLFormElement,
          );
```